src/components/v5/frame/PageLayout/PageLayout.tsx

```diff
@@ -61,7 +61,7 @@ const PageLayout: FC<PropsWithChildren<PageLayoutProps>> = ({
             <section className="w-full overflow-auto">
               <div
                 className={clsx(
-                  'mx-auto max-w-[1144px] px-6 pb-4 pt-6 md:pt-0',
+                  'mx-auto max-w-[calc(1144px+3rem)] px-6 pb-4 pt-6 md:pt-0',
```

Comment:
this change was made because the design width not include paddings: <img width="944" alt="image" src="https://github.com/user-attachments/assets/4d95f346-0fd6-43eb-ac9d-6bf6c0f90707">
But our container width was, so actual container width was 1096px:
<img width="866" alt="image" src="https://github.com/user-attachments/assets/ded1c324-c9be-42ce-929c-27f3b2931f1a">